### PR TITLE
Fix update group

### PR DESCRIPTION
**Pull request by nidovic**
fix date when update.

### CODE REVIEW

File: apps/server/src/services/packages/packages.service.ts [hunk 1]
```diff
@@ -65,7 +63,15 @@ export class PackagesService {
...
-    const fond = await this.packageRepository.findOne({id: idPackage});
+    const fond = await this.packageRepository.findOne(
+      { id: idPackage },
+      {
+        where: {
+          id: idPackage
+        },
+        relations: ['group']
+      }
+    );
```
Comment (Tadjaur) [inline apps/server/src/services/packages/packages.service.ts:74]: let use this declaration `findOne(id?: string | number | Date | ObjectID, options?: FindOneOptions<Entity>)`.
it's more efficient so update your implementation to:
```suggestion
    const fond = await this.packageRepository.findOne( idPackage,  { relations: ['group'] } );
```

**Review by Tadjaur — CHANGES_REQUESTED** [review]
Overall look good, please fix the suggestion before getting approval.

File: apps/server/src/services/packages/packages.service.ts (hunk 1 above)
Comment (nidovic) [reply apps/server/src/services/packages/packages.service.ts:74]: FindOne refuses relationship, and since we are looking for by id there is no difference.

Comment (Tadjaur) [reply apps/server/src/services/packages/packages.service.ts:74]: So as long it works, we don't care about the rest ie optimization and readability. is that it?

**Pull request merged**